pyopencl/__init__.py

```diff
@@ -1552,10 +1552,10 @@ def __init__(self, ctx, size, alignment, flags, *, _interface,
 
 # {{{ create_some_context
 
-def create_some_context(interactive: Optional[bool] = None,
-                        answers: Optional[List[str]] = None) -> Context:
+def choose_device(interactive: Optional[bool] = None,
+                  answers: Optional[List[str]] = None) -> List[Device]:
```

Comment:
Naming wise, it's weird that this returns a list and it's called `choose_device`?

Reply:
Yeah, we could rename it to `choose_devices` or so. My only concern was that most (?) people will just choose 1 device...